figures/client-reconnect-sequence.mmd

```diff
@@ -0,0 +1,9 @@
+sequenceDiagram
+    Actor Client
+    Client->>+OTT: Joins Through Balancer
+    OTT-->>+Client: Join Event
+    OTT->>-OTT: Stores Reference to Client
+    Client-->>+OTT: Disconnects
+    OTT->>OTT: Check Client WebSocket Disconnect Code
```

Comment:
I don't think the activation should happen here on disconnect (it should probably be a deactivation from the previous activation), and it should probably be a `-x` or `--x` link here.